dbt_project.yml

```diff
@@ -35,9 +35,10 @@ vars:
     resume: "{{ ref('stg_lever__resume') }}"
 
     lever_using_requisitions: true
+    lever_using_posting_tag: true
```

Comment:
I find it confusing that in the `source` codebase, we're only supposed to add these variables if they are false, but in the modeling codebase they exist by default? Happy to adapt both PRs to a consistent standard

My loosely-held opinions:
- Explicit is better than implicit: If a config variable can be set, most simple just to include with the default value in the dbt_project.yml. 
- D.R.Y - I would remove the default `,True` when the variable is referenced: `config(enabled=var('lever_using_posting_tag', True)` ➡️ `config(enabled=var('lever_using_posting_tag')`. That way, the value `true` is only typed once, in the config file, instead of being repeated in each file. The desire to "have default be `true`" can be enforced in one (and only one) place.

Reply:
Yeah, I can see how this is confusing.  Our standard is to explicitly define variables in all dbt_project.yml files, so that users (especially dbt beginners) have an example to follow when modifying their own dbt_project.yml file.

Not having this in the source package is just an oversight on our side.

Reply:
Makes sense, I'll update https://github.com/fivetran/dbt_lever_source/pull/5.

 Mind approving/requesting changes on this PR?